### PR TITLE
asset: add samples for ListAssets v1p5beta1

### LINKED ISSUE #1560: asset: add samples for ListAssets v1p5beta1

### PR DESCRIPTION
**Pull request by Larittic-GG**
Fixes #1560

### CODE REVIEW

File: asset/quickstart/list-assets/main.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START asset_quickstart_list_assets]
+
+// Sample asset-quickstart list assets.
```
Comment (tbpg) [inline asset/quickstart/list-assets/main.go:17]:
```suggestion
// Sample list-assets lists assets.
```

The default binary name is the name of the directory it's in (or the last part of the module name).

File: asset/quickstart/list-assets/main.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
+	assetType := "storage.googleapis.com/Bucket"
+	req := &assetpb.ListAssetsRequest{
+		Parent:      fmt.Sprintf("projects/%s", projectID),
+		AssetTypes:  []string{assetType},
+		ContentType: assetpb.ContentType_RESOURCE,
+	}
+
+	// Call ListAssets API to get an asset iterator.
+	assetIterator := client.ListAssets(ctx, req)
```
Comment (tbpg) [inline asset/quickstart/list-assets/main.go:48]:
```suggestion
	it := client.ListAssets(ctx, req)
```

Following the pattern of https://github.com/googleapis/google-cloud-go/wiki/Iterator-Guidelines.

File: asset/quickstart/list-assets/main.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	// Traverse and print the listed assets in response.
+	for {
+		response, err := assetIterator.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			log.Fatal(err)
+		}
+		fmt.Print(response)
```
Comment (tbpg) [inline asset/quickstart/list-assets/main.go:59]:
```suggestion
		fmt.Println(response)
```

That way different assets are printed on separate lines.

File: asset/quickstart/list-assets/main.go (hunk 1 above)
Comment (Larittic-GG) [reply asset/quickstart/list-assets/main.go]: Gotcha. Updated.

File: asset/quickstart/list-assets/main.go (hunk 2 above)
Comment (Larittic-GG) [reply asset/quickstart/list-assets/main.go]: Done.

File: asset/quickstart/list-assets/main.go (hunk 3 above)
Comment (Larittic-GG) [reply asset/quickstart/list-assets/main.go]: Done. Thanks!

**Comment by tbpg** [comment]
For some reason, the tests are timing out when running on Kokoro. Any ideas why?

```
=== RUN   TestMain
panic: test timed out after 1h0m0s

goroutine 151 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1296 +0xfd
created by time.goFunc
	/usr/local/go/src/time/sleep.go:172 +0x44
...
golang.org/x/net/http2.readFrameHeader(0xc0001ea9d8, 0x9, 0x9, 0xb14a60, 0xc0002c2a80, 0x0, 0x0, 0xc000254570, 0x0)
	/go/pkg/mod/golang.org/x/net@v0.0.0-20200707034311-ab3426394381/http2/frame.go:237 +0x7b
golang.org/x/net/http2.(*Framer).ReadFrame(0xc0001ea9a0, 0xc000254570, 0xc000254500, 0x0, 0x0)
	/go/pkg/mod/golang.org/x/net@v0.0.0-20200707034311-ab3426394381/http2/frame.go:492 +0xa3
google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc0001fee00)
	/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/http2_client.go:1287 +0x17b
created by google.golang.org/grpc/internal/transport.newHTTP2Client
	/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/http2_client.go:309 +0xedf
FAIL	github.com/GoogleCloudPlatform/golang-samples/asset/quickstart/list-assets	3600.014s
```

**Comment by Larittic-GG** [comment]
I have no idea so far. My local tests all passed very quickly. The newly added code is completely similar to other APIs' sample code. It should only require two env param, "GOOGLE_CLOUD_PROJECT" and "GOOGLE_APPLICATION_CREDENTIALS". Does it run many times in a roll? Could it have hit quota issue?

> For some reason, the tests are timing out when running on Kokoro. Any ideas why?
> 
> ```
> === RUN   TestMain
> panic: test timed out after 1h0m0s
> 
> goroutine 151 [running]:
> testing.(*M).startAlarm.func1()
> 	/usr/local/go/src/testing/testing.go:1296 +0xfd
> created by time.goFunc
> 	/usr/local/go/src/time/sleep.go:172 +0x44
> ...
> golang.org/x/net/http2.readFrameHeader(0xc0001ea9d8, 0x9, 0x9, 0xb14a60, 0xc0002c2a80, 0x0, 0x0, 0xc000254570, 0x0)
> 	/go/pkg/mod/golang.org/x/net@v0.0.0-20200707034311-ab3426394381/http2/frame.go:237 +0x7b
> golang.org/x/net/http2.(*Framer).ReadFrame(0xc0001ea9a0, 0xc000254570, 0xc000254500, 0x0, 0x0)
> 	/go/pkg/mod/golang.org/x/net@v0.0.0-20200707034311-ab3426394381/http2/frame.go:492 +0xa3
> google.golang.org/grpc/internal/transport.(*http2Client).reader(0xc0001fee00)
> 	/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/http2_client.go:1287 +0x17b
> created by google.golang.org/grpc/internal/transport.newHTTP2Client
> 	/go/pkg/mod/google.golang.org/grpc@v1.30.0/internal/transport/http2_client.go:309 +0xedf
> FAIL	github.com/GoogleCloudPlatform/golang-samples/asset/quickstart/list-assets	3600.014s
> ```

**Comment by Larittic-GG** [comment]
The test is still succeeding in my local environment but timing out here in Kokoro Cl. I think it might be due to quota reasons. Where can I find the project id that runs the test? i.e., what is the environment variable "GOOGLE_CLOUD_PROJECT" for the Kokoro Cl test?

**Pull request merged**